dcat/index.html

```diff
@@ -12,6 +12,9 @@
 <body>
 
 <!-- Disclaimer -->
+    <aside class="ednote">
+    <p>This document presents advancements in the discussion made on DCAT 3 within DXWG group. It complements the DCAT 2 [[?VOCAB-DCAT-2-20200204]] with guidelines on Versioning and Dataset series (see sections <a href="#dataset-versions"></a> and <a href="#dataset-series"></a>). </p>
```

Comment:
I suggest changing 'complements' to 'extends'

Reply:
Done